packages/chrome/src/index.tsx

```diff
@@ -96,7 +104,7 @@ const App = () => {
     setColorMode,
   }
 
-  if (!context.theme) return false
+  if (!context.theme) return null
```

Comment:
Okay, I think I know where this FunctionComponent came from :D Probably no harm to leave it.